docs/cloud_testing.md

```diff
@@ -0,0 +1,41 @@
+5. Export an environment variable `CLUSTER_CONFIG_FILE` which points to the checked-out copy of `observability-test-environments/tests/environments/elastic_cloud_tf_gcp.yml`.
+6. Export a variable for your test run which differentiates your cluster from others. This value may be arbitrary and will just be used to ensure that your cluster does not overwrite another: `export BUILD_NUMBER=my-great-test`.
+6. From the `observability-test-environments/ansible` directory checked out in step 1, run `make create-cluster`. This will take several minutes to complete.
+7. After the cluster has been configured, a number of additional environment variables will need to be exported. They are located in a file which should now be present in `observability-test-environments/ansible/build/config_secrets.yml`. The file contains various paths to Vault keys which store the secrets themselves. Below [is a table showing](#config-secrets) which environment variables must be set, which Vault keys they are stored in and which values they are located within the YAML data structure returned by a Vault lookup. Following the table, ensure that allow variables listed in the first column of the table are exported.
+8. Export a variable called `BUILD_OPTS` which contains the flags which should be passed to the APM Integration Test suite. At a minimum, the following must be set: `--apm-server-url $APM_SERVER_URL --apm-server-secret-token $APM_SERVER_TOKEN`. To duplicate the values used by the CI, the following may also be added: `--no-elasticsearch --no-apm-server --no-kibana --no-apm-server-dashboards --no-apm-server-self-instrument`.
+9. Run the tests as desired. Various scripts for starting up tests may be found in [the scripts directory](../.ci/scripts/). For example, to run all tests, execute `all.sh` or to run just the Python tests, execute `python.sh`.
+10. Once testing has been concluded, please tear down your cloud environment by running `make destroy-cluster` from the `observability-test-environments/ansible` directory.
+
+#### Config secrets
+
+The following variables must be exported to your shell in order to point the APM Integration Test runner at the provisioned cloud instance.
+
+|ENV  |Config value for vault lookup|Value from Vault lookup|
+|-----|----------|------------------------------------------|
+|APM_SERVER_URL|k8s_vault_apm_def_secret|url|
+|APM_SERVER_TOKEN|k8s_vault_apm_def_secret|token|
+|ELASTIC_APM_SECRET_TOKEN|k8s_vault_apm_def_secret|url|
+|ES_URL|k8s_vault_elasticsearch_def_secret|url|
+|ES_USER|k8s_vault_elasticsearch_def_secret|username|
+|ES_PASS|k8s_vault_elasticsearch_def_secret|password|
+|KIBANA_URL|k8s_vault_kibana_def_secret|url|
+
+### Accessing your cluster
+
+After provisioning your cluster, examine `observability-test-environments/ansible/build/terraform/elastic_cloud/main.tf` to locate the name of your deployment.
```

Comment:
Idea: could we make the clusters to print out the info needed to access the cluster? maybe even listing the mandatory env vars in an _exportable_ format.

Reply:
it is printed in the Ansible output

Reply:
Sorry, I meant it while running it locally, not on CI. I was wearing the works-on-my-computer hat